<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_task">
      <property name="text">
       <string>任务</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_layer">
      <property name="text">
       <string>层数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_time">
      <property name="text">
       <string>时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_nozzle">
      <property name="text">
       <string>喷嘴温度</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_hotbed">
      <property name="text">
       <string>热床温度</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_stage">
      <property name="text">
       <string>状态</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_pull_all">
      <property name="text">
       <string>刷新全部信息</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
